dealership/contracts.py

```diff
@@ -1,12 +1,32 @@
 class Contract(object):
     pass
 
-
 class BuyContract(Contract):
     def __init__(self, vehicle, customer, monthly_payments):
-        pass
+        self.vehicle = vehicle 
+        self.customer = customer 
+        self.monthly_payments = monthly_payments
+
+    def total_value(self):
+        sale_price = self.vehicle.sale_price()
+        interest = self.vehicle.get_interest()
+        discount = 0.1 if self.customer.is_employee() == True else 0 
```

Comment:
I like this. It looks like Ruby. I perhaps will move the discount up as an atribute?